adyenofficial.php

```diff
@@ -944,6 +948,31 @@ public function displayForm()
             'hint' => $this->l('')
         );
 
+        if ($this->versionChecker->isPrestaShop16()) {
+            $fields_form[1]['form']['input'][] = array(
+                'type' => 'radio',
+                'label' => $this->l('Collapsable payment display'),
+                'name' => 'ADYEN_PAYMENT_DISPLAY_COLLAPSE',
+                'values' => array(
+                    array(
+                        'id' => 'enable',
+                        'value' => 1,
+                        'label' => $this->l('Enable')
+                    ),
+                    array(
+                        'id' => 'disable',
+                        'value' => 0,
+                        'label' => $this->l('Disable')
+                    )
+                ),
+                'is_bool' => true,
+                // phpcs:ignore Generic.Files.LineLength.TooLong
+                'hint' => 'Indicates whether the payment methods should be rendered in a list of collapsable items,
+                 during checkout',
+                'required' => true
```

Comment:
We are using a default 0 basically in the code so I think it's not needed to give it a required flag.

```suggestion
                'required' => false
```

Also I only found to have a default value if you set the configuration value in the installation and in the upgrade script. It would mean to set the ADYEN_PAYMENT_DISPLAY_COLLAPSE config to 0